### PR TITLE
March On map climate strike style iterations 

### PR DESCRIPTION
**Pull request by mikelmaron**

### CODE REVIEW

File: map.js [hunk 1]
```diff
@@ -417,10 +437,11 @@ const app = new Vue({
       if (!this.features.length) {
         return;
       }
+
       const loc = this.userLocation;
       const withDistance = this.features.map(function(feature) {
         const coords = feature.geometry.coordinates;
-        const distance = _this.distance(loc.latitude, loc.longitude, coords[1], coords[0]);
+        const distance = (typeof feature.properties.source != "undefined" && feature.properties.source != "actionnetwork" ? _this.distance(loc.latitude, loc.longitude, coords[1], coords[0]) : 100000);
```
Comment (jillh510) [inline map.js]: this will change all the maps; please only make the change if it's on the climate page.

File: map.js [hunk 2]
```diff
@@ -390,7 +400,16 @@ const app = new Vue({
         return;
       }
 
+      var isClimatePage = false;
```
Comment (jillh510) [inline map.js]: is there a better solution than having this code twice?

**Review by jillh510 — CHANGES_REQUESTED** [review]

File: map.js (hunk 1 above)
Comment (mikelmaron) [reply map.js]: changed

File: map.js (hunk 2 above)
Comment (mikelmaron) [reply map.js]: dedup'd

**Comment by mikelmaron** [comment]
@jillh510 addressed your comments, thanks. this code definitely needs refactoring -- will look into this on the other side of the climate strike event.

**Review by jillh510 — APPROVED** [review]

**Pull request merged**